src/content/docs/new-relic-solutions/best-practices-guides/cx-improve-page-load.mdx

```diff
@@ -75,17 +75,18 @@ Additional resources:
 * [Google's approach to LCP optimization](https://web.dev/optimize-lcp/).
 * [Lighthouse](https://developers.google.com/web/tools/lighthouse) is a tool by Google that runs a synthetic test against a specific page and provides a list of recommendations that include how to optimize CLS.
 
-## First input delay (FID)
+## Interaction to next paint (INP)
 
-First input delay (FID) is the time between when a user first interacts with a page to the time when the browser is able to respond. It's a field metric that varies based on real user behavior (results vary based on user impatience and action timing) but can be optimized by reducing total blocking time (TBT).
+Interaction to next paint (INP) is the time between when a user first interacts with a page to the time when the browser is able to respond. It's a field metric that varies based on real user behavior (results vary based on user impatience and action timing) but can be optimized by reducing total blocking time (TBT).
 
 To do this, you need to:
 * Break up long blocking tasks.
 * Optimize bloated JavaScript.
 * Look at moving logic server side and/or use web workers to run threads in the background.
```

Comment:
@CherylFrankenfield I didn't update the surrounding language originally written for FID. Is this still correct for INP?

Reply:
@ally-sassman It includes clicks, taps, and keyboard interactions with a page throughout its lifespan, and not just the first interaction. INP will be a score of the longest duration from a user's interactions.